common/pupil/screening.ts

```diff
@@ -27,3 +27,20 @@ export async function addPupilScreening(pupil: Pupil, screening: PupilScreeningI
 
     logger.info(`Added ${screening.status || 'pending'} screening for pupil ${pupil.id}`, screening);
 }
+
+export async function invalidatePupilScreening(screeningId: number) {
+    const screening = await prisma.pupil_screening.findUniqueOrThrow({ where: { id: screeningId } });
+
+    if (screening.invalidated) {
+        throw new RedundantError(`Pupil screening ${screeningId} is already invalidated`);
+    }
+
+    await prisma.pupil_screening.update({ where: { id: screeningId }, data: { invalidated: true } });
+    logger.info(`Invalidated pupil screening ${screeningId} for pupil ${screening.pupilId}`);
+}
+
+// Use this whenever you want to invalidate a pupil's screenings. A pupil should only have one valid screening at most; but we go sure that all are invalidated.
+export async function invalidateAllScreeningsOfPupil(pupilId: number) {
+    const { count } = await prisma.pupil_screening.updateMany({ where: { pupilId }, data: { invalidated: true } });
```

Comment:
This will make sure that we only update the entries that are valid. It doesn't change anything to the logic, but might release some pressure from the db.

```suggestion
    const { count } = await prisma.pupil_screening.updateMany({ where: { pupilId, invalidated: false }, data: { invalidated: true } });
```